src/resources/container.js

```diff
@@ -60,6 +61,62 @@ class ContainerResource {
     instantiateRenderEntity(options) {
         return null;
     }
+
+    /**
+     * Queries the list of available material variants.
+     *
+     * @returns {object} A map which maps a variant name as a string to the variant index
+     * @example
+     * // load a glb file and instantiate an entity with a render component based on it
+     * app.assets.loadFromUrl("statue.glb", "container", function (err, asset) {
+     *     var entity = asset.resource.instantiateRenderEntity({
+     *         castShadows: true
+     *     });
+     *     app.root.addChild(entity);
+     *     let materialVariants = Object.keys(asset.resource.getMaterialVariants());
```

Comment:
in the examples we so far stick to ES5, so use vars, but no let / const. Also, this should be JS and not TS, so avoid the types.